### PR TITLE
Floating IP Use Case

### PR DESCRIPTION
**Pull request by catlook**
closes issue #114

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-e4430c73f5/docs/cloud-networks/v2/developer-guide/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-7ef738bc54/docs/cloud-networks/v2/developer-guide/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-bc5bc5bcfb/docs/cloud-networks/v2/developer-guide/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-219eb04789/docs/cloud-networks/v2/developer-guide/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-b69e427b43/docs/cloud-networks/v2/developer-guide/)

File: api-docs/concepts/concepts-floating-ips.rst [hunk 1]
```diff
@@ -52,3 +60,5 @@ following caveats.
 - You must provision a new floating IP address instead of using an existing IP address,
   because current public IPv4 addresses cannot be converted to a floating IP address.
 - Floating IP addresses may be only IPv4. IPv6 is not supported.
+- Floating IP functionality is only available in the ```IAD`` datacenter, so all your
```
Comment (cyrichardson) [inline api-docs/concepts/concepts-floating-ips.rst]: Extra ` in front of IAD

File: api-docs/getting-started/configuring-network-variations/configuring-host-routes-curl.rst [hunk 2]
```diff
@@ -40,26 +40,26 @@ boot the server.
...
-   provision your server, or perform other related activities. In this example, the ID is 
+
+#. Copy the ``id`` value from the output. You will use this value when you create a subnet,
+   provision your server, or perform other related activities. In this example, the ID is
    ``a8fde776-e80f-47bb-a050-0c057d89afc3``, but use the ID from your response.
 
 .. _chr-creating-subnet-curl:
 
 Creating a subnet with host routes (cURL)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (cyrichardson) [inline api-docs/getting-started/configuring-network-variations/configuring-host-routes-curl.rst:51]: Do you want to shorten the heading marker ~ to be no longer than the title? (Check throughout?)

--rsp: nah...i did some prep for refactor as i was in here...but will seriously take that on next week!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-4ba39206f2/docs/cloud-networks/v2/developer-guide/).

**Comment by cyrichardson** [comment]
LGTM - merging.

**Pull request merged**